config/m4/compiler.m4

```diff
@@ -98,4 +98,5 @@ AS_IF([test "x$with_avx" != xyes],
 #
 # Set C++ optimization/debug flags to be the same as for C
 #
+CFLAGS="$CFLAGS -DCPU_FLAGS=\"$OPT_CFLAGS\""
```

Comment:
This flag (actually some name) is passed to COMPILER_OPTION. So as I see there is single way to set needed optimization target here.